client/Components/App.js

```diff
@@ -34,7 +46,17 @@ class App extends Component {
 const mapDispatchToProps = dispatch => {
   return {
     checkIfUserLoggedIn: () => dispatch(checkIfUserLoggedInThunk()),
-    fetchStretches: () => dispatch(fetchStretches())
+
+    // For testing purposes only, remove as needed
+    load: () => {
```

Comment:
Is this how we want to handle data? My understanding after last night was that we were going to put data on state based on how the user interacts with the application. Feel free to let me know if I misunderstood.

Reply:
This ``load`` function was created to test whether or not the Redux thunks were working or not and will be removed and/or altered. However, I think the ``App`` component should house some of these ``getAll...()`` thunks since we would need some data when any user first loads the application, such as ``Stretches`` and ``Cohorts``.

The rest of the thunks would best be housed in the top-level components of the particular view that needs the data.

Reply:
Got it, didn't see the testing comment, but all good now. Ready to merge!